stories/LoadingSpinner.stories.tsx

```diff
@@ -7,12 +7,15 @@ export default {
   component: LoadingSpinner,
   parameters: {
     docs: {
-      page: createDocsPage()
-    }
-  }
+      page: createDocsPage(),
+    },
+  },
 };
 
-export const Default = args => {
-  return <LoadingSpinner {...args} />;
+export const Default = (
+  args: Record<string, unknown> & {message?: string}
```

Comment:
maybe a nit, but looking at the LoadingSpinner proptypes, it's a string, so instead of making `message` optional, we could default the value to empty string (`{message: string = ""}`) otherwise looks good to me.

Reply:
Hmm, when I try to do that I get "A type literal property cannot have an initializer" error since I am essentially would be attempting to initialize the value of a type (I believe that is what the error is saying).  What I could do is render the `LoadingSpinner` component as
`<LoadingSpinner message={message ?? ""} />` to return an empty string if `message` is `undefined`.

Reply:
gosh, sorry... dunno what I was thinking there. yeah your solution is fine. also though the existing way would work too, so please disregard my original comment. more trouble than it's worth